mods/cnc/rules/defaults.yaml

```diff
@@ -831,6 +848,23 @@
 	MapEditorData:
 		Categories: Husk
 
+^CivHaystackOrIglooHusk:
+	Inherits: ^CivField
+	MapEditorData:
+		ExcludeTilesets: DESERT
+	-Tooltip:
+	GrantConditionOnTileSet@WINTER:
+		Condition: winter
+		TileSets: SNOW, WINTER
+	Tooltip@WINTER:
+		Name: Igloo (Destroyed)
+		RequiresCondition: winter
+		GenericVisibility: Enemy, Neutral
+		ShowOwnerRow: false
```

Comment:
Same question here, except this seems to be missing the `campaign-tooltips.yaml` override, which means the summer haystacks will use the internal Tooltip defaults. Is that intended?